code/modules/mob/living/simple_animal/hostile/terror_spiders/queen.dm

```diff
@@ -299,58 +294,75 @@
 		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/black, numlings)
 	else if(eggtype == TS_DESC_PURPLE)
 		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/purple, numlings)
+	else if(eggtype == TS_DESC_BROWN)
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/brown, numlings)
+	else if(eggtype == TS_DESC_MOTHER)
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/mother, numlings)
+	else if(eggtype == TS_DESC_PRINCE)
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/prince, numlings)
+	else if(eggtype == TS_DESC_PRINCESS)
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/queen/princess, numlings)
 	else
 		to_chat(src, "<span class='danger'>Unrecognized egg type.</span>")
 
+
+/mob/living/simple_animal/hostile/poison/terror_spider/queen/proc/ListAvailableEggTypes()
+	if(MinutesAlive() >= 20)
+		var/num_my_special = CountSpidersDetailed(TRUE, list(/mob/living/simple_animal/hostile/poison/terror_spider/mother, /mob/living/simple_animal/hostile/poison/terror_spider/prince, /mob/living/simple_animal/hostile/poison/terror_spider/queen/princess))
+		if(num_my_special == 0)
+			return list(TS_DESC_PRINCE, TS_DESC_PRINCESS) // Mother will be added to this list.... AFTER mothers are reworked.
+
+	var/list/valid_types = list(TS_DESC_RED, TS_DESC_GRAY, TS_DESC_GREEN)
+	var/num_browns = CountSpidersDetailed(FALSE, list(/mob/living/simple_animal/hostile/poison/terror_spider/brown))
+	if(num_browns < 2)
+		valid_types |= TS_DESC_BROWN
```

Comment:
+= here instead of |=.
|= will look through the list to see if the to be added item is already present or not. Which can't be the case here